pallets/automation-time/src/weights.rs

```diff
@@ -161,15 +161,17 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 	}
 	// Storage: System Account (r:2 w:2)
 	// Storage: ParachainStaking DelegatorState (r:1 w:1)
+	// Storage: ParachainStaking DelegationScheduledRequests (r:1 w:0)
+	// Storage: ParachainStaking DelegatorReserveToLockMigrations (r:1 w:0)
+	// Storage: Balances Locks (r:1 w:1)
 	// Storage: ParachainStaking CandidateInfo (r:1 w:1)
 	// Storage: ParachainStaking TopDelegations (r:1 w:1)
 	// Storage: ParachainStaking CandidatePool (r:1 w:1)
 	// Storage: ParachainStaking Total (r:1 w:1)
-	// Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
 	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
 	fn run_auto_compound_delegated_stake_task() -> Weight {
-		(74_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(9 as Weight))
+		(87_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(11 as Weight))
```

Comment:
Both new reads come from parachain-staking. One read to see if the delegator has scheduled their intent to leave. Another read to see if the delegators storage has been migrated to the new paradigm. The second read is temporary and will eventually removed.